Bottom: 736fc0a082efd215f025e71472f8e570c525fcf2
Top:    ca60414fd8ae42ce0248dbd82193726269ec2ba9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-24 10:36:07 +0000

Refresh of add-netbyname-to-the-select-ac

---

diff --git a/src/select.c b/src/select.c
index ebada31..b8d6081 100644
--- a/src/select.c
+++ b/src/select.c
@@ -1017,7 +1017,6 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
       InitConnectionLookup ();
       changed = ResetConnections (true, true) || changed;
 
-      SaveUndoSerialNumber ();
       MENU_LOOP (&PCB->NetlistLib);
       {
         Cardinal i;
@@ -1035,7 +1034,6 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
           }
       }
       END_LOOP;
-      RestoreUndoSerialNumber ();
 
       changed = SelectConnection (Flag) || changed;
       changed = ResetConnections (true, true) || changed;
